receiver/splunkhecreceiver/metadata.yaml

```diff
@@ -5,3 +5,6 @@ status:
   stability:
     beta: [metrics, logs]
   distributions: [contrib, splunk, sumo]
+  codeowners:
+    active: ["@atoulme"]
```

Comment:
Why have the field here? Could the info be pulled from the codeowners file? mostly im just asking to have a single source of truth

Reply:
See discussion in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23367, we eventually will want to generate .github/CODEOWNERS from this metadata.